.kitchen.yml

```diff
@@ -22,19 +25,18 @@ suites:
       aet:
         activemq:
           jmx_ip: '192.168.123.111'
-  - name: develop
+  - name: seleniumgrid
+    driver:
+      network:
+        - ["private_network", {ip: "192.168.123.112"}]
+      customize:
+        memory: 768
     run_list:
-      - recipe[aet::_develop]
-      - recipe[aet::default]
+      - recipe[java::default]
```

Comment:
Or should **java::default** recipe be executed by Selenium grid recipe in a similar way like for Karaf?